ondemand.osc.edu/apps/dashboard/views/widgets/_classroom.html

```diff
@@ -1,4 +1,4 @@
-<div class=" mx-auto" style="width: 400px;">
+<div class="border rounded mx-auto" style="width: 400px;">
```

Comment:
this doesn't really have anything to do with R/4.1.0 or ood_core, I just have been meaning to touch it up.